HOWTO.md

```diff
@@ -86,6 +86,66 @@ This will result into this module header:
                                                                    *your_module*
 ```
 
+## Config
+
+You can configure docgen on file basis. For example you can define how `functions` or `classes`
+are sorted.
+
+```lua
+---@config { ['function_order'] = 'ascending', ['class_order'] = 'descending' }
```

Comment:
Could also take a function that can do the sorting 😁

Reply:
I actually thought about the same :laughing: when implementing it, I will keep the current version tho, for simplicity